.github/workflows/k8s-docker-build-push.yml

```diff
@@ -129,6 +129,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       KUBECTL_MAJOR_MINOR_VERSION: ${{ needs.get-version-number.outputs.KUBECTL_MAJOR_MINOR_VERSION }}
+      GCLOUD_CLI_VERSION: ${{ needs.get-version-number.outputs.GCLOUD_CLI_VERSION }}

```

Comment:
This env var although passed is only passed into the windows docker build. Not the Linux (amd64/arm64) ones. Are we ok with the latest always being installed in Linux? Or should we specify the version using a build arg?

Reply:
Forgot to remove